tensorboard/components/tf_storage/storage.ts

```diff
@@ -184,13 +213,12 @@ function makeBindings<T>(fromString: (string) => T, toString: (T) => string): {
       // if this is a localStorage property, we always synchronize the value
```

Comment:
Now that the initializer isn't syncing the hash value to the property if useLocalStorage is true, this comment is stale.